calendar/modules/listener/constants.bal

```diff
@@ -43,4 +43,6 @@ const string HTTP_ERROR_MSG = "Error occurred while getting the HTTP response :
 const string ERR_EXTRACTING_ERROR_MSG = "Error occured while extracting errors from payload.";
 const string JSON_ACCESSING_ERROR_MSG = "Error occurred while accessing the JSON payload of the response.";
 const string ERR_WATCH_RESPONSE = "Error occurred while constructing WatchResponse record.";
+const string ERR_CHANNEL_REGISTRATION = "Unable to register new channel.";
+const string ERR_SCHEDULE = "Unable to schedule next subscription.";
```

Comment:
subscription renewal?

Reply:
Fixed in bb0c08e9a7f7da0b076f584b770b90e7d7a06aa4